ctapipe/io/tests/test_zfits.py

```diff
@@ -0,0 +1,33 @@
+from pkg_resources import resource_filename
+import os
+
```

Comment:
This test module needs to be skipped if protozfitsreader is not installed:

```python
import pytest
pytest.importorskip("protozfitsreader")
```

Reply:
thanks .. done 👍